src/main/java/frc/robot/subsystems/Shooter.java

```diff
@@ -1,33 +1,54 @@
 /* ==================================================
- * Authors:
+ * Authors: Jason Wang
  *
  * --------------------------------------------------
- * Description:
- *
+ * Description: Shooter.java contains the functions to control the shooter.
  * ================================================== */
 
 package frc.robot.subsystems;
 
 import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import edu.wpi.first.wpilibj.motorcontrol.PWMTalonSRX;
 
 public class Shooter extends SubsystemBase
 {
   // HARDWARE //
-
-  // declare motor controllers, solenoids, and other hardware here
-
-  // CONSTANTS //
-
-
-
-  // CONSTRUCTOR //
-
+  private PWMTalonSRX motor;
+  public double speed;
```

Comment:
We don't need this variable anymore because we are always going to pass in the speed using the spin() function.